test/binaries/test-libexif-0.6.21.c

```diff
@@ -0,0 +1,9 @@
+#include <stdio.h>
+
+int main() {
+	printf("This program is designed to test the cve-bin-tool checker.");
+	printf("It outputs a few strings normally associated with libcurl 7.59.0.");
```

Comment:
why there is a libcurl here?

Reply:
I took another file as template and just appended my text with sed, my bad. Just pushed the changes.